docs/legacy/getting-started-apm-server.asciidoc

```diff
@@ -1,60 +1,175 @@
 [[getting-started-apm-server]]
-== Getting started with APM Server
+== Get started with APM Server (self managed)
 
 ++++
-<titleabbrev>Get started</titleabbrev>
+<titleabbrev>Get started (Self managed)</titleabbrev>
 ++++
 
+TIP: The easiest way to get started with Elastic APM is by using our
+{ess-product}[hosted {es} Service] on {ecloud}.
+The {es} Service is available on AWS, GCP, and Azure.
+See <<apm-quick-start>> to get started in minutes.
+
+// TODO: MOVE THIS
 IMPORTANT: Starting in version 8.0.0, {fleet} uses the APM integration to set up and manage APM index templates,
 {ilm-init} policies, and ingest pipelines. APM Server will only send data to {es} _after_ the APM integration has been installed.
 
-The easiest way to get started with Elastic APM is by using our
-{ess-product}[hosted {es} Service] on {ecloud}.
-The {es} Service is available on AWS, GCP, and Azure,
-and automatically configures APM Server to work with {es} and {kib}.
+The APM Server binary receives performance data from your APM agents,
+validates and processes it, and then transforms the data into {es} documents.
+If you're on this page, then you've chosen to self-manage the Elastic Stack,
+and you now must decide how to run and configure the APM Server.
+There are two options, and the components required are different for each:
+
+* **<<setup-apm-server-binary,APM Server binary>>**
+* **<<setup-fleet-managed-apm,Fleet-managed APM Server>>**
+// * **<<setup-apm-server-ea,Standalone Elastic Agent-managed APM Server>>**
 
 [float]
-=== Hosted {es} Service
+[[setup-apm-server-binary]]
+=== APM Server binary
+
+Install, configure, and run the APM Server binary wherever you need it.
+Depending on your use-case, APM Server can be run centrally or on edge machines.
```

Comment:
I'd remove this sentence, as in the decision tree we suggest to go with Fleet managed for running it on the edge machine.

Reply:
Good point